compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -290,6 +290,8 @@
     DUPLICATE_KEY_IN_RECORD_LITERAL("duplicate.key.in.record.literal"),
     DUPLICATE_KEY_IN_TABLE_LITERAL("duplicate.key.in.table.literal"),
     DUPLICATE_KEY_IN_RECORD_LITERAL_SPREAD_OP("duplicate.key.in.record.literal.spread.op"),
+    INCLUSIVE_RECORD_LITERAL_WITH_SPREAD_OP("inclusive.record.literal.with.spread.op"),
```

Comment:
Saying "inclusive record literal" is not quite right, right? I think it should be "inclusive record type" or with maps "inclusive mapping type".